frappe/email/doctype/email_account/email_account.py

```diff
@@ -578,7 +578,7 @@ def process_mail(messages, append_to=None):
 				for folder in self.imap_folder:
 					if email_server.select_imap_folder(folder.folder_name):
 						email_server.settings["uid_validity"] = folder.uidvalidity
-						messages = email_server.get_messages(folder=f'"{folder.folder_name}"') or {}
+						messages = email_server.get_messages(folder=f'{folder.folder_name}') or {}
```

Comment:
This won't work if folder name have space in it Like "My Folder"

Reply:
Thanks to you, I discovered another problem with folders, if the folders are encoded other than UTF-8
I'm working on fixing this issue.